MyGym Website/js/custom.js

```diff
@@ -0,0 +1,31 @@
+
+  $(function () {
+
+    // MENU
+    $('.navbar-collapse a').on('click',function(){
+      $(".navbar-collapse").collapse('hide');
+    });
+
+    // AOS ANIMATION
+    AOS.init({
+      disable: 'mobile',
+      duration: 800,
+      anchorPlacement: 'center-bottom'
+    });
+
+
+    // SMOOTHSCROLL NAVBAR
+    $(function() {
+      $('.navbar a, .hero-text a').on('click', function(event) {
+        var $anchor = $(this);
+        $('html, body').stop().animate({
+            scrollTop: $($anchor.attr('href')).offset().top - 49
+        }, 1000);
+        event.preventDefault();
+      });
+    });    
+  });
+
+
+    
+
```

Comment:
Use a JS beautifier to increase the readability.
Here use this link -- https://codebeautify.org/jsviewer

Reply:
sure!!